application.properties

```diff
@@ -1 +1,3 @@
-jwt.secret = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiWV9.PjmN-JugVug"
\ No newline at end of file
+jwt.secret = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiWV9.PjmN-JugVug"
```

Comment:
I wonder if we should commit this? Maybe we should leave it blank and have it filled locally.

Reply:
Mmmm or we can store somewhere else... Database?